src/main/java/com/gmail/goosius/siegewar/utils/SiegeWarWarningsUtil.java

```diff
@@ -0,0 +1,46 @@
+package com.gmail.goosius.siegewar.utils;
+
+import com.gmail.goosius.siegewar.Messaging;
+import com.gmail.goosius.siegewar.enums.SiegeWarPermissionNodes;
+import com.gmail.goosius.siegewar.settings.SiegeWarSettings;
+import com.palmergames.bukkit.towny.TownyAPI;
+import com.palmergames.bukkit.towny.object.Translatable;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+public class SiegeWarWarningsUtil {
+    
+    /**
+     * Send bad configs warnings
+     *
+     * @param sender the CommandSender to send the message to
+     */
+    public static void sendBadConfigsWarnings(CommandSender sender) {
```

Comment:
This should be renamed so that it is clear it might not be sending a warning message all of the time, ie: sendWarningIfConfigBad().

Reply:
Done